Setup/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
```

Comment:
Look into combining what you have with this existing file https://github.com/retaildevcrews/helium-csharp/blob/main/docs/codespaces/Dockerfile. So we'll only have one file just for dev containers.

Reply:
Done